src/main/java/uk/gov/justice/hmpps/prison/service/AppointmentsService.java

```diff
@@ -17,6 +17,7 @@
 import uk.gov.justice.hmpps.prison.api.model.bulkappointments.Repeat;
 import uk.gov.justice.hmpps.prison.api.support.TimeSlot;
 import uk.gov.justice.hmpps.prison.core.HasWriteScope;
+import uk.gov.justice.hmpps.prison.core.ProxyUser;
```

Comment:
Is this needed

Reply:
No, I put it in then realised the controller had the annotation already. Will remove now.